Sources/MessageCollectionViewCell.swift

```diff
@@ -0,0 +1,89 @@
+/*
+ MIT License
+ 
+ Copyright (c) 2017 MessageKit
+ 
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+ 
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+ 
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+ */
+
+import UIKit
+
+class MessageCollectionViewCell: UICollectionViewCell {
+    
+    // MARK: - Properties
+    
+    let messageContainerView: UIView = {
+        
+        let messageContainerView = UIView()
+        messageContainerView.backgroundColor = .green
+        messageContainerView.layer.cornerRadius = 12.0
+        messageContainerView.layer.masksToBounds = true
+        return messageContainerView
+    }()
+    
+    let avatarImageView: UIImageView = {
+        
+        let avatarImageView = UIImageView()
+        avatarImageView.backgroundColor = .red
+        return avatarImageView
+    }()
+    
+    // MARK: - Initializer
+    
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupSubviews()
+        backgroundColor = .purple
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Methods
+    
+    func setupSubviews() {
+        addSubview(messageContainerView)
+        addSubview(avatarImageView)
+    }
+    
+    override func apply(_ layoutAttributes: UICollectionViewLayoutAttributes) {
+        guard let attributes = layoutAttributes as? MessagesCollectionViewLayoutAttributes else { return }
+        print(attributes.avatarSize)
```

Comment:
For print statements, you should probably forward them to a print statement that only works in `DEBUG` mode.